homeassistant/components/nightscout/__init__.py

```diff
@@ -0,0 +1,67 @@
+"""The Nightscout integration."""
+import asyncio
+import logging
+
+from py_nightscout import Api as NightscoutAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import SLOW_UPDATE_WARNING
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS = ["sensor"]
+_API_TIMEOUT = SLOW_UPDATE_WARNING - 1
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Nightscout component."""
+    hass.data.setdefault(DOMAIN, {})
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Nightscout from a config entry."""
+    host = entry.data[CONF_HOST]
```

Comment:
looking at the docs you created it seems this is a URL with scheme and potentially port, so CONF_URL may be a better fit here as CONF_HOST is mostly used for hostnames and in combination with CONF_PORT and CONF_SSL